src/main/java/org/truffleruby/core/exception/CoreExceptions.java

```diff
@@ -597,10 +606,28 @@ public DynamicObject noMethodError(String message, Object receiver, String name,
         return Layouts.NO_METHOD_ERROR.createNoMethodError(
                 context.getCoreLibrary().getNoMethodErrorFactory(),
                 messageString,
+                null,
+                backtrace,
+                receiver,
+                context.getSymbolTable().getSymbol(name),
+                argsArray);
+    }
+
+    @TruffleBoundary
+    public DynamicObject noMethodError(DynamicObject formatter, Object receiver, String name, Object[] args, Node currentNode) {
+        final DynamicObject argsArray = createArray(context, args, args.length);
+        DynamicObject exceptionClass = context.getCoreLibrary().getNoMethodErrorClass();
```

Comment:
Might as well add `final` here for consistency in new code (but also above while we are at it).